### DISCUSSION TITLE
Viewing Traces in Zipkin Through SkyWalking

### DISCUSSION THREAD

**Discussion by abhishekacharya828** [apache/skywalking#12278] [General]
I want to see the trace requests in Zipkin through the SkyWalking server. I have also connected SkyWalking with Zipkin, but I don't know how to see the traces in Zipkin. Could you guide me ?

**Comment by wu-sheng** [comment]
You could follow this, https://skywalking.apache.org/docs/main/latest/en/setup/backend/zipkin-trace/#lens-ui

**Reply by abhishekacharya828** [reply]
Yes, I followed this, but my problem is that I am unable to show the trace  in Zipkin. I have already set up Zipkin in the SkyWalking UI but cannot see the trace  inside it. Can you tell me how I can achieve this ?

![Screenshot from 2024-05-31 10-19-43](https://github.com/apache/skywalking/assets/134144944/f4e9031c-7e4f-4ae4-9ce7-fdcaf2d070bf)

**Reply by wu-sheng** [reply]
I don't know, maybe not received? You should check dataflow.

**Reply by abhishekacharya828** [reply]
How i check data flow ?
